payments/schema/mutations.py

```diff
@@ -554,9 +554,27 @@ def mutate_and_get_payload(cls, root, info, **input):
                         ),
                         "payment_url": payment_url,
                     }
-                    send_notification(
-                        email, NotificationType.ORDER_APPROVED.value, context, language
-                    )
+
+                    if order.order_type == OrderType.NEW_BERTH_ORDER:
```

Comment:
Could we extract function for cleaner code here:
`notification_type = get_notification_type(order.order_type)`
?